app/src/main/java/edu/uw/npctracker/core/Model.kt

```diff
@@ -0,0 +1,29 @@
+package edu.uw.npctracker.core
+
+/*
+ * @author owendm
+ * A model manages generated characters
+ */
+interface Model {
+
+    /**
+     * @return The number of characters currently in the model
+     */
+    fun size(): Int
+
+    /**
+     * @param character the character to add to the model, if is already in model does nothing
+     */
+    fun plusCharacter(character: Character)
```

Comment:
Do you think we could rename this to `addCharacter`?
```suggestion
    fun addCharacter(character: Character)
```

Reply:
I don't actually think so. [Look here](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/plus-element.html)